index.html

```diff
@@ -5,17 +5,34 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="https://fonts.googleapis.com/css2?family=Merriweather+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
-    <link
-      rel="stylesheet"
-      href="node_modules/font-awesome/css/font-awesome.min.css"
-    />
+    <link rel="stylesheet" href="node_modules/@fortawesome/fontawesome-free/css/all.min.css">
     <link rel="stylesheet" href="assets/styles/css/reset.css" />
     <link rel="stylesheet" href="assets/styles/css/style.css" />
     <title>Food & Wine</title>
 </head>
 <body>
     <header>
-        <!-- write header here -->
+        <div class="fw-hlogo">
+          <div class="fw-hcontacts"> <!-- Absolute block with contacts -->
+            <div class="fw-hcontactsblock">
+              <div><h4><i class="fas fa-map-marker-alt"></i> 4578 Marmoa Road, Glasgow D04 89GR</h4></div>
+              <div><h4><i class="fas fa-phone"></i> +1(800)123 1334  <i class="far fa-clock"></i>  10:00 am - 11:00 pm</h4></div>
+            </div>
+          </div>
+          <div class="fw-hlogoblock">
+            <img src="./assets/images/logov1.png" alt="Logo">
+          </div>
+          <nav class="fw-hnaw"> <!-- Navigation -->
+            <div class="fw-hnavitem"><a href="#">HOME</a></div>
```

Comment:
char **h** is redundant in class names
besides this extra 'h' lowers readability, in such case it is better to separate it with '-'
`fw-h-naw` `fw-h-nav-item`

Reply:
Char **h** added to the class name to avoid conflicts with other sections, where we can use class-name "fw-contacts" or "fw-logoblock". Therefore, it was added  "-h-" to the class names.